www/navbar.html

```diff
@@ -14,7 +14,13 @@
     <div class="collapse navbar-collapse" id="hail-navbar-collapse">
       <ul class="nav navbar-nav navbar-right">
         <li id="home" class="nav-item active"><a href="index.html">HOME</a></li>
-        <li id="docs" class="nav-item"><a href="docs/stable/index.html">DOCS</a></li>
+        <li id="docs" class="nav-item">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">DOCS<span class="caret"></span></a>
+          <ul class="dropdown-menu dropdown-menu-left">
+            <li><a class="dropdown-item" href="docs/stable/index.html">Stable (0.1)</a></li>
```

Comment:
This also needs to be changed in `python/hail/docs/_templates/navbar_hail.html`.

I can't remember why we needed two different navbar template files.